### ISSUE TITLE
iPhone app PIN unlock times out too quickly (and the timeout period is not documented)

### ISSUE THREAD

**Issue by gsharratt** [bitwarden/mobile#2862]
### Steps To Reproduce

1. Configure the BW iPhone app with Unlock with PIN code = ON; Face ID = OFF.
2. Wait some period of time, e.g., 36 hours
3. Open the app
4. There is a high probability that the app does NOT offer unlocking via PIN, only unlocking via master password

### Expected Result

1. Configure the BW iPhone app with Unlock with PIN code = ON; Face ID = OFF
2. **Further configure the period of time X (hours or days) that the PIN is to remain active (i.e., how long before it times out)**
4. Wait some period of time Y (hours or days)
5. Open the app
6a. If Y < X, the app DOES offer unlocking via PIN
6b. If Y > X, the app does NOT offer unlocking via PIN, only unlocking via master password

### Actual Result

1. Configure the BW iPhone app with Unlock with PIN code = ON; Face ID = OFF.
2. **(There is no way to configure the period of time X (hours or days) that the PIN is to remain active (i.e., how long before it times out))**
3. Wait some period of time Y (hours or days) -- e.g., 36 hours
4. Open the app 
5. There is a high probability that the app does NOT offer unlocking via PIN, only unlocking via master password

### Additional Context

- I've never figured out whether the trigger for the PIN timing out (i.e., no longer being offered) is (a) a particular number of hours/days or (b) some random event.  I'm pretty sure I never get the PIN prompt after 1.5 to 2 days.
- The BW documentation does not state how long PIN unlock is supposed to remain active.
- IMO PIN unlock should be configurable to remain active for a period of time ranging from 1 hour to (at most) 1 month.

### Operating System

iOS

### Operating System Version

the problem has been present for at least a year (so many different versions of iOS/iPadOS)

### Device

iPhone 13 mini, iPad mini 6

### Build Version

the problem has been present for at least a year (so many different versions of BW)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by jtodddd** [comment]
Hi there,

This is expected behavior as when you log out of your mobile app, your unlock with PIN settings will be wiped and you will need to re-enable Unlock with PIN (https://bitwarden.com/help/unlock-with-pin/#tab-mobile-DeLAPOlNpLH3MaKazVTfU)

We use GitHub issues as a place to track bugs and other development related issues. If you have any questions please write us back using our contact form, so we can continue troubleshooting: https://bitwarden.com/contact/ 

You can include a link to this issue in the message content.

Alternatively, you can also search for an answer in our help documentation (https://bitwarden.com/help/) or get help from other Bitwarden users on our community forums (https://community.bitwarden.com/c/support/). 

The issue here will be closed.

Thanks!

**Comment by gsharratt** [comment]
> ... when you log out of your mobile app, your unlock with PIN settings will be wiped and you will need to re-enable Unlock with PIN (https://bitwarden.com/help/unlock-with-pin/#tab-mobile-DeLAPOlNpLH3MaKazVTfU)

That's not what's happening.  I did not log out of the app.  Please review the repro again.

**Comment by gsharratt** [comment]
@jtodddd
I wanted to be sure I didn't miss anything so I just did another search of BW help pages.  
All I found was the point you referenced:
> When you log out of your mobile app, your unlock with PIN settings will be wiped and you will need to re-enable Unlock with PIN.
So to be clear, in the repro above I never logged out after logging in.  I simply switched to a different app or turned off the mobile device. 
Unlock with PIN works fine for some period of time, but as mentioned ^^ that period of time is apparently random (roughly a day, or two at most, I think) and not defined in any way in the help files.
Hence this issue.
